lib/screens/productsNew/newUI.dart

```diff
@@ -14,22 +14,22 @@ import 'package:mvp/bloc/productsapi_bloc.dart';
 import 'package:mvp/classes/storeProducts_box.dart';
 import 'package:mvp/constants/themeColours.dart';
 import 'package:mvp/domain/product_repository.dart';
-// import 'package:mvp/models/newCart.dart';
 import 'package:mvp/models/storeProducts.dart';
 import 'package:mvp/screens/common/cartIcon.dart';
 import 'package:mvp/screens/productsNew/details.dart';
 import 'package:mvp/sizeconfig/sizeconfig.dart';
-// import 'package:provider/provider.dart';
 import 'package:shimmer/shimmer.dart';
 
 class Category {
   final String name;
+  final String resname;
```

Comment:
need better variable name, otherwise cannot understand what exactly it signifies or how it is being used

Reply:
ok I will change it